### PR TITLE
Create prepo.scss

### PR DESCRIPTION
**Pull request by liqwiz**
#### Hi! What is your PR about?

- [x] Add or edit a skin
- [ ] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

File: skins/prepo.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  #topnav a {
+    color: white !important;
+  }
+
+  #topnav button {
+    border-color: white;
+    color: white;
+  }
```
Comment (gokaiorg) [inline skins/prepo.scss:22]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
```suggestion
```

Comment (gokaiorg) [reply skins/prepo.scss:22]: Capture d’écran 2022-01-25 à 09 23 00

**Review by gokaiorg — CHANGES_REQUESTED** [review]
You will also need to list your skin on this file:
https://github.com/snapshot-labs/snapshot-spaces/blob/master/skins/index.js

**Comment by liqwiz** [comment]
Thanks for the review @gokaiorg, I've fixed the theme and updated the index file.

File: skins/prepo.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: white;
+
+  #topnav {
+    background-color: #454699;
```
Comment (gokaiorg) [inline skins/prepo.scss:12]: Please add the background colour globally, you can edit the #topnav border only. 
```suggestion
```

File: skins/prepo.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  #topnav a, #topnav .text-gray {
+    color: white !important;
+  }
+
+  #topnav button {
+    border-color: white;
+    color: white;
+  }
```
Comment (gokaiorg) [inline skins/prepo.scss:23]:
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by liqwiz** [comment]
> Please add the background colour globally, you can edit the #topnav border only.

Hi @gokaiorg, thanks for your response. 

We copied the Yearn Skin almost exactly (https://github.com/snapshot-labs/snapshot-spaces/blob/master/skins/yearn.scss), only changing the primary brand color. Is no longer a valid approach?

**Comment by gokaiorg** [comment]
@bonustrack Can the types of Yearn styles be reproduced?

**Comment by liqwiz** [comment]
Hi @gokaiorg, just following up here. 

Is it not OK to modify the header like Yearn or do you have some other suggestions how I can achieve this effect?

Thanks.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**